test_scripts/Polices/Related_HMI_API/ATF_OnAllowSDLFunctionality_allowed_false_without_device.lua

```diff
@@ -0,0 +1,79 @@
+---------------------------------------------------------------------------------------------
+-- Description: 
+--     1. Preconditions: App is registered
+--     2. Steps: Activate App, send SDL.OnAllowSDLFunctionality with 'allowed=false' and without 'device' to HMI
+-- Requirement summary: 
+--     [Policy] "EXTERNAL_PROPRIETARY" flow: Related HMI API
+--     [Policies]: OnAllowSDLFunctionality with 'allowed=false' and without 'device' param from HMI
+--
+-- Expected result:
+--     In case PoliciesManager receives SDL.OnAllowSDLFunctionality with 'allowed=false' and without 'device' param from HMI, PoliciesManager must record 
+--     all of currently registered devices as NOT consented in Local PT ("device_data" - > "<device_id_1>", "<device_id_2>", etc. - >"user_consent_records"- > "device" sub-section).
+---------------------------------------------------------------------------------------------
+--[[ General Settings for configuration ]]
+Test = require('connecttest') 
+require('cardinalities')
+require('mobile_session')
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+--[[ Local Functions ]]
+local function get_id_value() 
+  local sql_select = "sqlite3 " .. tostring(SDLStoragePath) .. "policy.sqlite \"SELECT id FROM functional_group WHERE name = "BaseBeforeDataConsent"\""
+    local aHandle = assert( io.popen( sql_select , 'r'))
+    sql_output = aHandle:read( '*l' )   
+    local retvalue = tonumber(sql_output)    
```

Comment:
@vvvakulenko According to script coding style you should use for local variables snake_case.